application.properties

```diff
@@ -0,0 +1,20 @@
+#
+# @(#) application.properties Copyright (c) 2019 Jala Foundation.
+# 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+# All rights reserved.
+#
+# This software is the confidential and proprietary information of
+# Jala Foundation, ("Confidential Information").  You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jala Foundation.
+#
+
+## CONFIGURE THE APP CLIENT.
+
+app.client_id=
+app.client_secret=
+app.redirect_uri=
+app.user_name=
+app.password=
+app.token=
```

Comment:
Please rename the name of the key:
app.token
As there are two kinds of tokens, It would be better if this is named as:
app.securityToken

Reply:
Ok I think is more describe.